### DISCUSSION TITLE
How to add a row to a collection using the REST API?

### DISCUSSION THREAD

**Discussion by glowkeeper** [directus/directus#3997] [Archive]
I can't see from the docs how you insert a new row into a collection via the REST API. 

Say I have a collection called _foo_ with a column named _bar_, and I want to add a row (item) to that collection. I'm expecting to be able to do something like this:

```
      const fooData = {
         bar: "directus rox"
      }
      fetch("https://myserver:port/foo/items, {
        method: 'POST',
        headers: {
          'Content-Type': 'application/json'
        },
        body: JSON.stringify(fooData)
      })
 ```
 
 But the docs suggest I just `POST /items` - if that's the case, how does directus know to which collection I'm posting? I must be missing something :(

**Comment by rijkvanzanten** [comment]
Ah good catch, that's supposed to be `POST /items/:collection/` instead of just `POST /items` 🙂  Fixing that now..

The rest of your example looks right to me 👍🏻

**Reply by rijkvanzanten** [reply]
All set: https://docs.directus.io/reference/api/rest/items/

**Reply by glowkeeper** [reply]
Aha! Phew - thanks :)

...I suppose I could've just tried it :)
